docs/api/urls.md

```diff
@@ -98,9 +98,9 @@ Flag multiple URLs in the same context with a context manager.
     When a fallback view is given it *must* take the same arguments as the flagged view.
 
 ```python
-with flagged_urls('MY_FLAG') as url:
+with flagged_urls('MY_FLAG') as furl:
     flagged_url_patterns = [
-        url(^'a-url/', view_requiring_flag),
+        furl(^'a-url/', view_requiring_flag),
```

Comment:
:+1: This helps distinguish flagged URLs in a potentially long list governed by a context manager.

Reply:
It's also that using `url` there is harmful in Python 2 — the `url` is in the local scope of the `with`, not the block it contains, so all subsequent `url` calls will be flagged.

So, yeah.